user_docs/en/userGuide.t2t

```diff
@@ -1855,9 +1855,14 @@ Some of these features may be incomplete.
 The following options exist: 
 - "Report details in browse mode": enables reporting if an object has details in browse mode.
 Reporting the summary of those details can be done by assigning a gesture using the [Input Gestures dialog #InputGestures].
-- "Report aria-description always": enables reporting of aria-descriptions.
-Note: aria-description only applies to the web, and is only a subset of the sources for "description".
--
+- "Report aria-description always":

```

Comment:
Seems good to me. Whoever is interested in knowing what this option does will have the key words to search for more information if needed.